tools/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": "../tsconfig.base.json",
+  "compilerOptions": {
+    "noEmit": true
+  },
+  "include": [],
+  "files": [],
+  "references": [
```

Comment:
environment scoped TS config - this is the future for every convergence package (will mitigate issues like this -> https://github.com/microsoft/fluentui/issues/17101)